crates/pulsar-core/src/event.rs

```diff
@@ -196,12 +199,18 @@ pub enum Payload {
     Fork {
         ppid: i32,
         namespaces: Namespaces,
```

Comment:
probably `Namespaces` should be included in the header via process_tracker and here should be a `skip(validatron)`. in this way the only reason of having it in the fork event is an optimization to avoid multiple readings from `/proc`  . what do you think?

Reply:
same thing for exec event